### ISSUE TITLE
Cite your project

### ISSUE THREAD

**Issue by upponup** [tensorflow/addons#2717]
Hi, first of all, thanks for that great work!

I just wanted to clarify what is the correct way to reference your repository when writing an article? Do you have any ```CITATION.cff``` or ```BibTex``` file?

**Comment by bhack** [comment]
No, we still don't have one with the Zenodo entry as in TF https://github.com/tensorflow/tensorflow/blob/master/CITATION.cff.
